guvnor-ala/guvnor-ala-distribution/src/test/java/org/guvnor/ala/services/tests/RuntimeEndpointsTestIT.java

```diff
@@ -43,88 +43,57 @@
     @Ignore
     public void checkService() {
         Client client = ClientBuilder.newClient();
-        WebTarget target = client.target(APP_URL);
-        ResteasyWebTarget restEasyTarget = (ResteasyWebTarget) target;
-        RuntimeProvisioningService proxy = restEasyTarget.proxy(RuntimeProvisioningService.class);
+        WebTarget target = client.target( APP_URL );
```

Comment:
Good catch, well be fixed. On the other hand I can confirm I've been using the correct code styling template.